test/server/cards/03_TWI/events/NowThereAreTwoOfThem.spec.ts

```diff
@@ -0,0 +1,69 @@
+describe('Now There are Two of Them', function() {
+    integration(function(contextRef) {
+        describe('Now There are Two of Them\'s ability -', function() {
+            it('should allow you to play a non-Vehicle unit from your hand that shares a Trait with the unit you control for 5 less, if you have only one unit in play', function() {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        hand: ['now-there-are-two-of-them', 'zuckuss#bounty-hunter-for-hire', 'cad-bane#hostage-taker', 'plo-koon#kohtoyah', 'fetts-firespray#pursuing-the-bounty'],
+                        groundArena: ['greedo#slow-on-the-draw'],
+                        base: { card: 'administrators-tower', damage: 0 },
+                        leader: 'qira#i-alone-survived',
+                        resources: 5
+                    }
+                });
+
+                const { context } = contextRef;
+
+                // You can select Zuckuss and Cad Bane. Zuckuss is free to play.
+                context.player1.clickCard(context.nowThereAreTwoOfThem);
+                expect(context.player1.exhaustedResourceCount).toBe(3);
+                expect(context.player1).toBeAbleToSelectExactly([context.zuckuss, context.cadBane]);
+                context.player1.clickCard(context.zuckuss);
+                expect(context.zuckuss).toBeInZone('groundArena');
+                expect(context.player1.exhaustedResourceCount).toBe(3);
+
+                // You can't select Cad Bane if do not have enough resources.
```

Comment:
This check could be done simpler by just including an 8-cost underworld card like Jabba's Rancor or the Chewie unit and confirming that it isn't targetable in the above situation. Then we wouldn't need this part at all